libkbfs/folder_branch_ops.go

```diff
@@ -1978,8 +1978,21 @@ func (fbo *folderBranchOps) finalizeMDWriteLocked(ctx context.Context,
 		return err
 	}
 
-	// Archive the old, unref'd blocks
-	fbo.fbm.archiveUnrefBlocks(irmd.ReadOnly())
+	// Archive the old, unref'd blocks if journaling is off.
+	doArchive := true
+	if jServer, err := GetJournalServer(fbo.config); err == nil {
```

Comment:
We do this in at least one other place, right? Maybe we should have a `TLFJournalEnabled(TlfID) bool` helper function